node/src/components/consensus/protocols/highway.rs

```diff
@@ -304,7 +304,7 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
         // register the proposal before the meter is aware that a new round has started, and it
         // will reject the proposal.
         if vv.is_proposal() {
-            // unwraps are safe, as if value is `Some`, this is already a unit
+            // unwraps are safe, as if value is `Some`, this is already a vote
```

Comment:
Should we be renaming it back? @afck Just changed it to _unit_.

Reply:
That is insanely fast turnaround

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/521/commits/252ccb96053d8793432ecb94a0f927c0100d63ac